hyperlink/_url.py

```diff
@@ -970,7 +950,9 @@ def from_text(cls, text):
                             raise URLParseError('port must not be empty')
                         raise URLParseError('expected integer for port, not %r'
                                             % port_str)
-        family, host = parse_host(host)
+        # _, host = parse_host(host)
+        if host:
+            host = host.lstrip('[').rstrip(']')
```

Comment:
Heh. There should be an online swear jar for anyone who ever says "just" in a code review :)